src/util/file-filter.js

```diff
@@ -99,7 +96,7 @@ export class FileFilter {
    * file in the folder that is being archived.
    */
   wantFile(filePath: string): boolean {
-    const resolvedPath = this.resolve(filePath);
+    const resolvedPath = path.resolve(filePath);
```

Comment:
I want to remove some "concatenation magic". I'll put the details in next comment.